packages/bpk-component-banner-alert/src/AnimateAndFade-test.js

```diff
@@ -27,7 +28,7 @@ const message = (
 describe('AnimateAndFade', () => {
   it('should render correctly', () => {
     const tree = renderer
-      .create(<AnimateAndFade>{message}</AnimateAndFade>)
+      .create(<AnimateAndFade show>{message}</AnimateAndFade>)
```

Comment:
Why did you have to add the show prop here?

Reply:
Because this was previously a bug. Show was required, but not specified

Reply:
Nice one!

Reply:
Well actually this should be `show={false}` to maintain the previous behaviour, but since show is ever only considered in `componentWillReceiveProps` it has no bearing on this test